docs/Introduction.Android.md

```diff
@@ -72,7 +72,7 @@ android {
   defaultConfig {
       // ...
       testBuildType System.getProperty('testBuildType', 'debug')  // This will later be used to control the test apk build type
-      testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+      testInstrumentationRunner 'com.wix.detox.DetoxJUnitRunner'
```

Comment:
I can't help but wonder whether posting this in the docs could be a bit too soon. Shouldn't we get this through some mileage first?

Reply:
Agree.